be/src/vec/exprs/vcompound_pred.h

```diff
@@ -158,12 +158,15 @@ class VCompoundPred : public VectorizedFnCall {
         if (_can_fast_execute && fast_execute(context, block, result_column_id)) {
             return Status::OK();
         }
-        if (get_num_children() == 1 || !_all_child_is_compound_and_not_const()) {
+        if (get_num_children() == 1 || !_all_child_is_not_const()) {
```

Comment:
`!_all_child_is_not_const()` is hard to understand. It is equal to `has_const_child()`